content/docs/reference/microcontrollers/arduino-leonardo.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Arduino Leonardo"
+weight: 3
+---
+
+The [Arduino Leonardo](https://store.arduino.cc/collections/boards/products/arduino-leonardo-with-headers) is based on the AVR [ATmega32u4](https://www.microchip.com/en-us/product/ATmega32u4) microcontroller.
+
+Note: the AVR backend of LLVM is still experimental so you may encounter bugs.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | Not yet |
+| SPI       | YES | Not yet |
+| I2C       | YES | Not yet |
+| ADC       | YES | YES |
+| PWM       | YES | Not yet |
+| USBDevice | YES | Not yet |
+
+## Pins
+
+| Pin               | Hardware pin | Alternative names |
+| ----------------- | ------------ | ----------------- |
+| `D0`              | `PD2`        |                   |
+| `D1`              | `PD3`        |                   |
+| `D2`              | `PD1`        |                   |
+| `D3`              | `PD0`        |                   |
+| `D4`              | `PD4`        |                   |
+| `D5`              | `PC6`        |                   |
+| `D6`              | `PD7`        |                   |
+| `D7`              | `PE6`        |                   |
+| `D8`              | `PB4`        |                   |
+| `D9`              | `PB5`        |                   |
+| `D10`             | `PB6`        |                   |
+| `D11`             | `PB7`        |                   |
+| `D12`             | `PD6`        |                   |
+| `D13`             | `PC7`        | `LED`             |
+| `ADC0`            | `PF7`        |                   |
+| `ADC1`            | `PF6`        |                   |
+| `ADC2`            | `PF5`        |                   |
+| `ADC3`            | `PF4`        |                   |
+| `ADC4`            | `PF1`        |                   |
+| `ADC5`            | `PF0`        |                   |
+
+## Machine Package Docs
+
+[Documentation for the machine package for the Arduino Leonardo](../machine/arduino-leonardo)
+
+## Installing dependencies
+
+The Arduino Leonardo needs a few extra dependencies to work, for example, if you get an error like this:
+
+```text
+/usr/lib/gcc/avr/5.4.0/../../../avr/bin/ld: cannot find -lm
+/usr/lib/gcc/avr/5.4.0/../../../avr/bin/ld: cannot find -lc
+collect2: error: ld returned 1 exit status
+```
```

Comment:
These errors won't happen anymore with the dev branch. Please take a look here: https://github.com/tinygo-org/tinygo-site/pull/321

Reply:
Fixed! thanks for the heads up.